tests/graph/test_utils.py

```diff
@@ -0,0 +1,12 @@
+from threading import Thread
+
+from pydantic_graph._utils import get_event_loop
+
+
+def test_get_event_loop_in_thread():
+    def get_and_close_event_loop():
+        event_loop = get_event_loop()
+        event_loop.close()
+
+    thread = Thread(target=get_and_close_event_loop)
+    thread.start()
```

Comment:
do you need to join this thread after creating it? Or does that happen automatically